### PR TITLE
feat: toast

### PR DESCRIPTION
**Pull request by shashidwivedi**
Add Toast component

image

### CODE REVIEW

File: components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  background-color: var(--color-dark);
+  border-radius: 4px;
+  color: var(--color-light);
+  justify-content: flex-start;
+  display: flex;
+  align-items: center;
+  opacity: 0;
+}
+
+.toast--open .toast-body {
```
Comment (vaibhav-mahalle) [inline components/toast/toast.css:27]: you can follow the consistent naming pattern here you have used "toast--open" but you could have named it "toast-open"

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some improvement please look into it.

File: components/toast/toast.css (hunk 1 above)
Comment (shashidwivedi) [reply components/toast/toast.css:27]: Yes Vaibhav, will keep this in mind. Thanks for pointing it out

**Review by rittik-ghosh — APPROVED** [review]
Please look into the comments maid by vaibhav.
Everything else looks great

**Comment by shashidwivedi** [comment]
> Please look into the comments maid by vaibhav. Everything else looks great

Yes, will adjust the naming here for other elements also. Actually, the **--open** in **toast--open** is a modifier, that's why I have put two -- in the naming. Will refactor it later to make the naming more clear. Thanks!

**Pull request merged**